src/service/service.js

```diff
@@ -2,10 +2,9 @@ import { WORDS_COUNT } from '../constants/constants';
 import { getRandomIndexInRange } from '../helpers';
 import { words } from '../data/data';
 
-
-const getCorrectWords = () => {
+const getWords = () => {
     const index = getRandomIndexInRange(WORDS_COUNT);
     return [...words[index]];
 };
 
-export { getCorrectWords };
+export { getWords };
```

Comment:
I missed it earlier but `service.js` is not the best name, it's like naming a component file `component.js`.
How about `get-words.js`?